### PR TITLE
Icons are clickable & Trash icon prompts confirmation pop-up

### PR DESCRIPTION
**Pull request by vithura14**
Add clickable edit, trash and pin icons (all icons respects the color theme).
Specifically the trash icon prompts a pop-up, when clicked, confirming user's decision to delete a message. 

image

image

### CODE REVIEW

File: src/components/MessageIcons.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    { text: "Yes", onPress: () => console.log("OK Pressed") }
+  ]
+);
+class MessageIcons extends Component {
+
+
+  render() {
+    return (
+      <MessageIconsContainer>
+              <ClickEdit onPress={this.onButtonPress}>
```
Comment (yasfahm) [inline src/components/MessageIcons.js]: fix indentation of ClickEdit, ClickTrash and ClickPin

File: src/components/MessageIcons.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      </MessageIconsContainer>
+      
+    ); 
+  }
+
+  }
+
+  
+  // STYLED-COMPONENTS
+  const MessageIconsContainer = styled.View`
```
Comment (yasfahm) [inline src/components/MessageIcons.js]: remove space

File: src/components/MessageIcons.js (hunk 1 above)
Comment (vithura14) [reply src/components/MessageIcons.js]: Done!

File: src/components/MessageIcons.js (hunk 2 above)
Comment (vithura14) [reply src/components/MessageIcons.js]: Done!

**Review by yasfahm — APPROVED** [review]

**Comment by yasfahm** [comment]
there are some minor Codacy issues, you can merge after fixing those!

**Pull request merged**